i18n/locales/en.ts

```diff
@@ -879,6 +879,31 @@ const translations = {
     verify_signature_one: {
       title: 'Verify the signature',
     },
+    verify_signature_five: {
+      title: 'Verify the signature',
+      heading: 'Verify the signature!',
+      success: "You've done it! You've verified Satoshi's signature!",
+      paragraph_one:
+        'At this point we have everything we need to do some ECDSA math.',
+      paragraph_two:
+        'The ECDSA signature verification algorithm is explained <Link className="underline" href="https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm#Signature_verification_algorithm" target="_blank">here</Link> and <Link className="underline" href="https://www.secg.org/sec1-v2.pdf" target="_blank">here</Link> (Go to page 46, section 4.1.4).',
```

Comment:
We might just want to add a page number fragement here to link to 4.1.4 in the pdf `#page=52`